src/v/debug_bundle/debug_bundle_service.cc

```diff
@@ -299,12 +299,37 @@ service::rpk_debug_bundle_status() {
       .cerr = _rpk_process->_cerr.copy()};
 }
 
-ss::future<result<std::filesystem::path>> service::rpk_debug_bundle_path() {
+ss::future<result<std::filesystem::path>>
+service::rpk_debug_bundle_path(job_id_t job_id) {
     if (ss::this_shard_id() != service_shard) {
         co_return co_await container().invoke_on(
-          service_shard, [](service& s) { return s.rpk_debug_bundle_path(); });
+          service_shard,
+          [job_id](service& s) { return s.rpk_debug_bundle_path(job_id); });
     }
-    co_return error_info(error_code::debug_bundle_process_never_started);
+    auto status = process_status();
+    if (!status.has_value()) {
+        co_return error_info(error_code::debug_bundle_process_never_started);
+    }
+    switch (status.value()) {
+    case debug_bundle_status::running:
+        co_return error_info(error_code::debug_bundle_process_running);
+    case debug_bundle_status::success:
+        break;
+    case debug_bundle_status::error:
+        co_return error_info(error_code::process_failed);
+        break;
```

Comment:
nit: No need to break after a return (like in the first case)

Reply:
yup ypu